x-pack/plugins/security_solution/public/resolver/view/panels/process_details.tsx

```diff
@@ -51,7 +52,7 @@ export const ProcessDetails = memo(function ProcessDetails({
 }) {
   const processName = event.eventName(processEvent);
   const entityId = event.entityId(processEvent);
-  const isProcessTerminated = useSelector(selectors.isProcessTerminated)(entityId);
+  const isProcessTerminated = useShallowEqualSelector(selectors.isProcessTerminated)(entityId);
```

Comment:
Unrelated to this PR but: `isProcessTerminated` actually should be a parametric selector in order to work as expected.